### PR TITLE
Reduce storage requirements for webrtc canaries

### PR DESCRIPTION
**Pull request by lherman-cs**
Reduce storage requirements by pruning library dependencies and source codes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: webrtc-c/canary/CMakeLists.txt [hunk 1]
```diff
@@ -6,14 +6,16 @@ set(CMAKE_CXX_STANDARD 11)
...
+set(ENABLE_TESTING OFF CACHE BOOL "Disable building tests for AWS SDK" FORCE)
+set(BUILD_ONLY "monitoring;logs" CACHE STRING "Tell AWS SDK to only build monitoring and logs" FORCE)
+set(BUILD_SHARED_LIBS OFF CACHE BOOL "Enable static build for AWS SDK" FORCE)
+set(BUILD_STATIC_LIBS ON CACHE BOOL "Enable static build for KVS Webrtc SDK" FORCE)
 
 FetchContent_Declare(
   webrtc
   GIT_REPOSITORY https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c
-  GIT_TAG        master
+  GIT_TAG        35fc3ea5f2132613a8439da41fdfc1b5c052c858
```
Comment (disa6302) [inline webrtc-c/canary/CMakeLists.txt:18]: In this case, wouldn't we have to keep changing the git tag every time a new commit gets merged in the webrtc repo?

Comment (lherman-cs) [reply webrtc-c/canary/CMakeLists.txt:18]: Well, it's only when we want to update it. Right now, our canary's script doesn't lock in git hash. So, I figure we should do this for now.

Comment (lherman-cs) [reply webrtc-c/canary/CMakeLists.txt:18]: I just want to make sure that we retrieve the version that has the static build fix.

Comment (disa6302) [reply webrtc-c/canary/CMakeLists.txt:18]: Got it. So, this will be changed to master ultimately, correct?

**Review by MushMal — APPROVED** [review]

**Review by disa6302 — APPROVED** [review]

File: webrtc-c/canary/CMakeLists.txt (hunk 1 above)
Comment (lherman-cs) [reply webrtc-c/canary/CMakeLists.txt:18]: Yea.. as soon as I finish updating the Groovy script to lock in the version hash.

**Pull request merged**